file-formats/srvc/type/zif_aff_srvc_v1.intf.abap

```diff
@@ -1,29 +1,33 @@
 INTERFACE zif_aff_srvc_v1
   PUBLIC.
   "! <p class="shorttext">Consumer Type</p>
-  "! Consumer Type
+  "! Consumer type
   "! $values {@link zif_aff_srvc_v1.data:co_cons_type}
   TYPES ty_cons_type TYPE c LENGTH 2.
 
   CONSTANTS:
     BEGIN OF co_cons_type,
       "! <p class="shorttext">OData</p>
       "! OData
+      "! $enumValue 'OData'
       odata       TYPE ty_cons_type VALUE '01',
       "! <p class="shorttext">Web Service</p>
-      "! Web Service
+      "! Web service
+      "! $enumValue 'WebService'
       web_service TYPE ty_cons_type VALUE '02',
       "! <p class="shorttext">RFC</p>
       "! RFC
+      "! $enumValue 'RFC'
       rfc         TYPE ty_cons_type VALUE '03',
       "! <p class="shorttext">HTTP</p>
       "! HTTP
+      "! $enumValue 'HTTP'
       http        TYPE ty_cons_type VALUE '04',
     END OF co_cons_type.
 
   TYPES:
-    "! <p class="shorttext">Consumer Connection Property</p>
-    "! Consumer Connection Property
+    "! <p class="shorttext">Connection Property</p>
```

Comment:
Usually we add "Details" at the end of an array entry.

```suggestion
    "! <p class="shorttext">Connection Property Details</p>
```

Reply:
Added details to the property name